### ISSUE TITLE
Change description in Chrome Web Store

### ISSUE THREAD

**Issue by MartinLichtblau** [acarabott/tabToWindow#16]
I didn't find your addon at first and tried out some other e.g. [Open-as-Popup](https://chrome.google.com/webstore/detail/open-as-popup/ncppfjladdkdaemaghochfikpmghbcpc/reviews?utm_source=chrome-app-launcher-info-dialog) and [TabToPopup](https://chrome.google.com/webstore/detail/tabtopopup/bpnnnahgjgehfimcicpjmjidcoghhfpf?utm_source=chrome-app-launcher-info-dialog). 

Both are not very well made and a joke compared to yours. But I didn't find yours because you don't have the keyword "Popup" written in it, you call it "Pop Tab" and so most people can't find it. 

So in short: I suggest you to change the description so that people who look for such an extension, which moves the current tab into a popup, can find it. In my case, I wanted to save space and furthermore I like the minimal UI of the Chrome Popups.

**Comment by acarabott** [comment]
Thanks, great feedback. Have updated it :+1:
